sourcemash/frontend/templates/splash.tpl

```diff
@@ -0,0 +1,6 @@
+<div class="row">
+  <div class="col s12 valign-wrapper">
+    <img src="/static/solologo.svg">
+    <img src="/static/logotext.svg">
+  </div>
+</div>
```

Comment:
I won't be able to edit an SVG file in Pixelmator. We'd need KO for that I think.